tests/shell/Makefile

```diff
@@ -6,9 +6,6 @@ USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
 
-# Use a terminal that does not introduce extra characters into the stream.
-RIOT_TERMINAL ?= socat
```

Comment:
This will cause problems when testing long lines, see https://github.com/RIOT-OS/RIOT/pull/10635/files#diff-16e3498e34b06780aaf14222a95484a5R91

Reply:
This seems like a bad bug…

Currently the other issue with `pyterm` is that it exits when `ctrl+c` is sent.